.classpath

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Spent some time here, unfortunately, eclipse is not great at parameterizing `.classpaths`. They recommend maven building the `.classpath` file, etc. Removing for now.